demos/graph-classification/dgcnn-graph-classification.ipynb

```diff
@@ -0,0 +1,761 @@
+{
```

Comment:
<p>Tiny thing, but I think an "inline list" like this should be either separated by a comma or start with a colon:</p><p> </p><blockquote>The experimental evaluation scheme in [1] does not specify some important details such as the regularisation used for the neural network layers, if a bias term is included, the weight initialization method used, and the batch size.&nbsp;</blockquote><p> </p><blockquote>The experimental evaluation scheme in [1] does not specify some important details such as: the regularisation used for the neural network layers; if a bias term is included; the weight initialization method used; and the batch size.&nbsp;</blockquote><p> </p><p>It took me a little bit to understand the grouping in this sentence as is, as it seemed like "The experimental ... network layers" was one entire clause and "if a bias term is included" was another separate one, rather than "The experimental ... important details" and "the regularisation ... layers" being separate clauses.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1319/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>